src/main/rst/03_webdriver.rst

```diff
@@ -960,6 +1037,11 @@ popup, you can access the alert with the following:
 
     $driver->get_alert_text();
     $driver->accept_alert();
+    
+.. code-block:: javascript
+
+	var alert = new new driver.switchTo().alert();
```

Comment:
also incorrect - it's not a constructor and can't be new'd.

    var alert = driver.switchTo().alert();